c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -444,7 +444,7 @@ compile-py-catalog: $(SERVER_LOCALISATION_FILES)
 
 $(PACKAGE)/locale/$(PACKAGE)-server.pot: $(SERVER_LOCALISATION_SOURCES_FILES) $(VENV_BIN)/pot-create lingua.cfg
 	$(VENV_BIN)/pot-create -c lingua.cfg -o $@ $(SERVER_LOCALISATION_SOURCES_FILES)
-	# removes the allways changed date line
+	# always remove date lines which have changed
```

Comment:
always is about changed not remove ...
=> ?
remove date lines which are always changing